po/fr.po

```diff
@@ -5884,33 +5882,33 @@ msgstr "Assumer oui à toutes les questions"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:42
 msgid "Show how much disk space will be used or freed"
-msgstr ""
+msgstr "Afficher l’espace disque qui sera utilisé ou libéré"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:43
-#, fuzzy
 msgid "Specify the format to be used by the search command"
-msgstr "Tester si apt doit se faire donner la sous-commande"
+msgstr "Préciser le format à utiliser par la commande search"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:44
 msgid "Set the release from which packages should be installed"
-msgstr ""
+msgstr "Préciser la distribution de laquelle installer les paquets"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:45
 msgid "Specify the order for the output from the search command"
-msgstr ""
+msgstr "Préciser l’ordre de la sortie de la commande search"
 
 #: /tmp/fish/implicit/share/completions/aptitude.fish:46
-#, fuzzy
 msgid "Set a configuration file option directly"
-msgstr "Spécifier un fichier de configuration"
+msgstr ""
+"Spécifier une option de configuration directement au lieu de l’inclure dans "
```

Comment:
“Spécifier une option de configuration directement” would be the direct translation from the English, the rest is added explanatory verbiage; not wrong as such, just not part of the original message.

Reply:
OK, I'll stick with the english, more concise version.